Index.html

```diff
@@ -38,17 +38,209 @@ <h2>
                 a project you need coded, don't hesitate
                 to contact me.
             </p>
-
+         <div class="icon-link">
             <ul>
                 <li><a href="#"><span class="fa fa-twitter"></span></a></li>
                 <li><a href="#"><span class="fa fa-angellist"></span></a></li>
                 <li><a href="#"><span class="fa fa-github"></span></a></li>
                 <li><a href="#"><span class="fa fa-linkedin"></span></a></li>
             </ul>
         </div>
+        </div>
+        </section>
+
+        <section class="Work">
+            
+            <h1>My Recent Work</h1>
+            <div class="grid-items">
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                    <div class="See-btn">
+                        
+                        <button type="button" class="see-project">See Project</button>
```

Comment:
- [ ] As I've mentioned in the summary, kindly remove the white space before the `<button>` tag.
- [ ] It looks like something is off with the button styling, especially when `hover`, `active`, and `pressed`. Check the image catalog below to ensure that these styles are captured for the buttons.
![original](https://user-images.githubusercontent.com/26856491/121516154-ced9b500-c9e5-11eb-8307-072aa03ad301.png)